Libplanet/Net/Swarm.cs

```diff
@@ -353,6 +327,51 @@ IEnumerator IEnumerable.GetEnumerator()
                 $"but {parsedMessage}");
         }
 
+        // TODO: Add TxIds
+        public IAsyncEnumerable<Block<T>> GetDataAsync<T>(
```

Comment:
Would it be directly used by library users?  To a library user, `IAsyncEnumerable<T>` can be unfamiliar.

Reply:
> Would it be directly used by library users? 

It was not decided at this stage.

> To a library user, IAsyncEnumerable<T> can be unfamiliar.

I agree. should we provide `IEnumerable<T>` version? (but, it may be synchronous.)